### ISSUE TITLE
Can't add a new Vuetify icon set

### ISSUE THREAD

**Issue by RebeccaStevens** [userquin/vuetify-nuxt-module#97]
I'm trying to follow [the guide on the docs site](https://vuetify-nuxt-module.netlify.app/guide/icons/unocss-preset-icons.html#adding-a-new-vuetify-icon-set) but it's not working for me. Would I be able to get some help?

<details>
<summary>I'm getting the error: <code>aliases</code> is not defined</summary>

```
[nuxt] [request error] [unhandled] [500] aliases is not defined
  at Module.iconsConfiguration (/@nuxt-vuetify-configuration/vuetify-icons-configuration:10:5)  
  at Module.configureIcons (./node_modules/.pnpm/vuetify-nuxt-module@0.5.7_rollup@2.79.1_typescript@5.2.2_vite@4.4.9_vue@3.3.4/node_modules/vuetify-nuxt-module/dist/runtime/plugins/icons.mjs:5:50)  
  at ./node_modules/.pnpm/vuetify-nuxt-module@0.5.7_rollup@2.79.1_typescript@5.2.2_vite@4.4.9_vue@3.3.4/node_modules/vuetify-nuxt-module/dist/runtime/plugins/vuetify-icons.mjs:7:27  
  at ./node_modules/.pnpm/nuxt@3.7.1_@types+node@18.17.15_eslint@8.49.0_rollup@2.79.1_sass@1.66.1_stylelint@15.10.3_typ_rb4vbjg7aaksx7nzfntqydkqsa/node_modules/nuxt/dist/app/nuxt.js:65:44  
  at fn (./node_modules/.pnpm/nuxt@3.7.1_@types+node@18.17.15_eslint@8.49.0_rollup@2.79.1_sass@1.66.1_stylelint@15.10.3_typ_rb4vbjg7aaksx7nzfntqydkqsa/node_modules/nuxt/dist/app/nuxt.js:162:44)  
  at Object.callAsync (./node_modules/.pnpm/unctx@2.3.1/node_modules/unctx/dist/index.mjs:68:55)  
  at ./node_modules/.pnpm/nuxt@3.7.1_@types+node@18.17.15_eslint@8.49.0_rollup@2.79.1_sass@1.66.1_stylelint@15.10.3_typ_rb4vbjg7aaksx7nzfntqydkqsa/node_modules/nuxt/dist/app/nuxt.js:164:56  
  at Object.runWithContext (./node_modules/.pnpm/@vue+runtime-core@3.3.4/node_modules/@vue/runtime-core/dist/runtime-core.cjs.js:3855:18)  
  at callWithNuxt (./node_modules/.pnpm/nuxt@3.7.1_@types+node@18.17.15_eslint@8.49.0_rollup@2.79.1_sass@1.66.1_stylelint@15.10.3_typ_rb4vbjg7aaksx7nzfntqydkqsa/node_modules/nuxt/dist/app/nuxt.js:164:24)  
  at Object.runWithContext (./node_modules/.pnpm/nuxt@3.7.1_@types+node@18.17.15_eslint@8.49.0_rollup@2.79.1_sass@1.66.1_stylelint@15.10.3_typ_rb4vbjg7aaksx7nzfntqydkqsa/node_modules/nuxt/dist/app/nuxt.js:34:29)
```

</details>

#### Here's my config files

```ts
// nuxt.config.ts
import { type NuxtConfig } from "nuxt/config";
export default {
  modules: [
    // ...
    "@unocss/nuxt",
    "vuetify-nuxt-module",
  ],
  vuetify: {
    moduleOptions: {
      styles: true,
    },
  },
  // ...
} satisfies NuxtConfig;

```

```ts
// vuetify.config.ts
import { type ExternalVuetifyOptions } from "vuetify-nuxt-module";
export default {
  icons: {
    defaultSet: "custom",
  },
} satisfies ExternalVuetifyOptions;
```

```ts
// plugins/vuetify.ts
// @unocss-include
import { mdi } from "vuetify/lib/iconsets/mdi.mjs";
export default defineNuxtPlugin((nuxtApp) => {
  nuxtApp.hook("vuetify:configuration", ({ vuetifyOptions }) => {
    vuetifyOptions.icons = {
      defaultSet: "mdi",
      sets: { mdi },
      aliases: {
        // ...
      },
    };
  });
});
```

#### On a sidenote

The types for the `vuetify:configuration` hook don't seem to be registering for me. The hook is definitely running though as if I add a `console.log` to it, the data is printed out.

![image](https://github.com/userquin/vuetify-nuxt-module/assets/7224206/fbde5b6b-9d87-4ed2-a5df-d04a6dfad0d3)

**Comment by userquin** [comment]
I'm checking what's wrong...

**Comment by userquin** [comment]
fixed in [v0.5.8](https://github.com/userquin/vuetify-nuxt-module/releases/tag/v0.5.8)
